index.html

```diff
@@ -0,0 +1,145 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Super SportCar Global Race</title>
+  <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-EVSTQN3/azprG1Anm3QDgpJLIm9Nao0Yz1ztcQTwFspd3yD65VohhpuuCOmLASjC" crossorigin="anonymous">
+  <link rel="stylesheet" href="Assets/sass/main.css">
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.14.0/css/all.min.css"/>
+</head>
+<body class="bg-light" id="body">
+  <div class="container-fluid">
+    <div class="row d-md-block social">
+      <ul class="d-flex align-items-center justify-content-end align-content-center">
+        <li class="social-icon d-flex"><a href="#"><i class="fab fa-facebook"></i></a></li>
+        <li class="social-icon d-flex"><a href="#"><i class="fab fa-twitter"></i></a></li>
+        <li class="social-icon d-flex"><a href="#">English</a></li>
+        <li class="social-icon d-flex"><a href="#">My Page</a></li>
+        <li class="social-icon d-flex"><a href="#">Logout</a></li>
+      </ul>
+    </div>
+    <header class="row justify-content-center">
+      <nav class="navbar flex-md-row navbar-expand-md navbar-light">
+        <a class="navbar-brand d-md-flex flex-md-row-reverse" href="index.html"> SSCGR<br>2021 <i class="fas fa-car"></i></a>
```

Comment:
- [ ] When I click the logo on the `about` page, I got a 404 error. Please, fix it. 🚗
![image](https://user-images.githubusercontent.com/20563252/124698203-f2d9ca80-dead-11eb-8742-3c09ff6481a2.png)